src/common.props

```diff
@@ -0,0 +1,20 @@
+<Project>
+  <PropertyGroup>
+    <TargetFrameworks>net451;netstandard1.6</TargetFrameworks>
+    <NetStandardImplicitPackageVersion>1.6.1-*</NetStandardImplicitPackageVersion>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+
+    <AssemblyOriginatorKeyFile>../../tools/Key.snk</AssemblyOriginatorKeyFile>
+    <SignAssembly>true</SignAssembly>
+    <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
+
+    <RepositoryType>git</RepositoryType>
+    <RepositoryUrl>https://github.com/aspnet/javascriptservices</RepositoryUrl>
+
+    <GenerateNeutralResourcesLanguageAttribute>true</GenerateNeutralResourcesLanguageAttribute>
```

Comment:
Thanks for the pointers. I've added a reference to `Internal.AspNetCore.Sdk` (in `common.props`) and moved a few files around to make it pretty much the same as the other ASP.NET Core projects.